todo-mvc-kitchensink/tests/functional/Page.ts

```diff
@@ -79,7 +83,10 @@ export default class Page {
 			.get('http://localhost:9000/_build/src/index.html')
 			.setFindTimeout(5000)
 			.findByCssSelector(this.selectors.newInput)
-			.setFindTimeout(100);
+			.setFindTimeout(100)
+			.then(() => {
+				return this.errorCollector.init()
```

Comment:
It did in the IDE, but it appears that we don't lint on `dojo build` so I missed the little green squiggle... Is that what we expect/want from `dojo build`?  Should it have an option to do linting?